packages/teams-js/src/internal/utils.ts

```diff
@@ -422,12 +422,12 @@ export function validateId(id: string, errorToThrow?: Error): void {
   }
 }
 
-export function validateUrl(url: URL, errorToThrow?: Error): void {
+export function validateUrl(url: URL, errorToThrow?: Error, enforceMaxLength: boolean = true): void {
```

Comment:
I honestly want to remove this check all together, because this might be an issue with Other API too.